sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapCameraManagerDelegate.kt

```diff
@@ -220,17 +214,18 @@ interface MapCameraManagerDelegate {
   /**
    * Changes the map view by any combination of center, zoom, bearing, and pitch, without an animated transition.
    * The map will retain its current values for any details not passed via the camera options argument.
+   * It is not guaranteed that the provided CameraOptions will be set, the map may apply constraints resulting in a
+   * different CameraState.
    *
-   * @param camera New camera options
+   * @param cameraOptions New camera options
    */
   fun setCamera(cameraOptions: CameraOptions)
 
   /**
-   * Returns the current camera options
+   * Returns the current camera state
    *
-   * @param padding Optional edge padding
    */
-  fun getCameraOptions(padding: EdgeInsets?): CameraOptions
+  fun getCameraState(): CameraState
```

Comment:
can we use a `val cameraState: CameraState` here? So that users would just use `mapboxMap.cameraState.bearing` for example?

Reply:
We can do it and also remove all the methods here like `getLan`, `getLon` etc. I believe they were introduced to avoid `!!` explicit casting. I also would advise removing `setBearing` method as it looks a bit strange all alone.
@mapbox/maps-android WDYT?

Reply:
sounds good to me